src/metrics.jl

```diff
@@ -0,0 +1,15 @@
+# patches to metrics.jl of Distances.jl
+const UnionMetrics = Distances.UnionMetrics
```

Comment:
codes in this file are used to change the dispatch priority introduced by Distances